api/categorisation.go

```diff
@@ -13,7 +13,7 @@ import (
 // if so, it returns the relevant information. If there are no default categorisations, it returns empty string
 // for default categorisation, and the original dimension name and label to persist instead.
 // returns (finalDimension, finalLabel, finalCategorisation, error)
-func (api *API) RetrieveDefaultCategorisation(dimension *model.Dimension, datasetName string) (string, string, string, error) {
+func (api *API) RetrieveDefaultCategorisation(dimension *model.Dimension, datasetName string) (finalDimension, finalLabel, finalCategorisation string, err error) {
 	ctx := context.Background()
```

Comment:
these vars don't seem to be set ? Also - https://go-review.googlesource.com/c/go/+/20024

Reply:
what do you mean in the first question? 

re: the article - it was highlighted by the linter possibly because there's so many of them, seemed to add value to help the understanding of what is actually being returned at a glance.

Reply:
so the named values are not set in the function

Reply:
ok, I get you now - have removed this change and added nolint.